airprint/airprint-generate.py

```diff
@@ -131,7 +132,11 @@ def generate(self):
                 tree.parse(StringIO(XML_TEMPLATE.replace('\n', '').replace('\r', '').replace('\t', '')))
 
                 name = tree.find('name')
-                name.text = 'AirPrint %s @ %%h' % (p)
+		
+                if self.descName:
+                   name.text = '%s' % (v['printer-info'])
+                else:
+                   name.text = 'Airprint %s @ %%h' % (p)
```

Comment:
Please keep `AirPrint` with capital P as before

Reply:
please update this one last thing and we're good to merge

Reply:
done.